src/coreclr-debug/install.ts

```diff
@@ -162,28 +162,28 @@ export class DebugInstaller
                 emitEntryPoint: true
             },
             dependencies: {
-                "Microsoft.VisualStudio.clrdbg": "14.0.25406-preview-3044032",
-                "Microsoft.VisualStudio.clrdbg.MIEngine": "14.0.30610-preview-1",
-                "Microsoft.VisualStudio.OpenDebugAD7": "1.0.20614-preview-2",
-                "NETStandard.Library": "1.5.0-rc2-24027",
+                "Microsoft.VisualStudio.clrdbg": "14.0.25429-preview-3095034",
+                "Microsoft.VisualStudio.clrdbg.MIEngine": "14.0.30628-preview-1",
```

Comment:
I noticed that I accidentally missed Wes's MIEngine commit from yesterday in this build. I kicked off a new build with that. We probably should take that too.